openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSettingLeftPanel.tsx

```diff
@@ -83,7 +87,13 @@ const GlobalSettingLeftPanel = () => {
   const onClick: MenuProps['onClick'] = (e) => {
     // As we are setting key as "category.option" and extracting here category and option
     const [category, option] = e.key.split('.');
-    history.push(getSettingPath(category, option));
+    if (option === GlobalSettingOptions.TEAMS) {
+      history.push(
+        `${getSettingPath(category, option)}/${TeamType.Organization}`
+      );
```

Comment:
https://github.com/open-metadata/OpenMetadata/blob/106f99f4c8940ef488829da061a7c7815359da75/openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts#L229
This would be a great fit hear!

Reply:
made suggested changes